docs/_steps/adding_editing_workflow_step.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Adding or editing workflow steps
+lang: en
+slug: adding-editing-steps
+order: 3
+---
+
+<div class='section-content'>
+
+When a builder adds (or later edits) your step in their workflow, your app will receive a `workflow_step_edit` action. The callback assigned to the `edit` property of the `WorkflowStep` configuration object passed in during instantiation will run when this action occurs.
+
+Whether a builder is adding or editing a step, you need to provide them with a special `workflow_step` modal — a workflow step configuration modal — where step-specific settings are chosen. Since the purpose of this modal is tied to a workflow step's configuration, it has more restrictions than typical modals—most notably, you cannot include `title​`, `submit​`, or `close`​ properties in the payload. By default, the `callback_id` used for this modal will be the same as that of the workflow step.
+
+Within the `edit` callback, the `configure()` utility can be used to easily open your step's configuration modal by passing in an object with your view's `blocks`. To disable configuration save before certain conditions are met, pass in `submit_disabled` with a value of `true`.
```

Comment:
To disable saving the configuration?

Reply:
I'm interpreting this as a suggestion for language, rather than a point of clarification -- if so, will update to:

> To disable saving the configuration before certain conditions are met, pass in `submit_disabled` with a value of `true`